scripts/check_watch_statements.sh

```diff
@@ -0,0 +1,7 @@
+if grep -r "@watch" tests/functional/spec; then
```

Comment:
May as well check for `.only` and `@focus` as well?

Reply:
Done